plugin.cpp

```diff
@@ -406,6 +415,27 @@ void plugin_reconfigure(PLUGIN_HANDLE *handle, string &newConfig)
     opcua->reconfigure(config);
 }
 
+/**
+ * Setpoint control write operation
+ */
+bool plugin_write(PLUGIN_HANDLE *handle, string name, string value)
+{
+    OPCUA *opcua = (OPCUA *)handle;
+
+    if (!opcua)
+        return false;
+    return opcua->write(name, value);
+}
+
+/**
+ * Setpoint control operation. None are supported by s2opcua
+ */
+bool plugin_operation(PLUGIN_HANDLE *handle, string operation, int parameterCount, PLUGIN_PARAMETER parameters[])
+{
+    Logger::getLogger()->warn("OPC UA do not support Setpoint control operation");
```

Comment:
OPC UA do not support Setpoint control operation -> OPC UA plugin does not support Setpoint control operation